packages/components/src/menu-items-choice/types.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Internal dependencies
+ */
+import type { ShortcutProps } from '../shortcut/types';
+
+export type MenuItemsChoiceProps = {
+	/**
+	 * Array of choices
```

Comment:
Just as a style consistency thing, we end all descriptions with a period.

```suggestion
	 * Array of choices.
```